src/jesse_cli.erl

```diff
@@ -90,7 +90,7 @@ jesse_run(JsonInstance, Schema, Schemata) ->
   {ok, _} = application:ensure_all_started(jesse),
   ok = add_schemata(Schemata),
   {ok, JsonInstanceBinary} = file:read_file(JsonInstance),
-  JsonInstanceJsx = jsx:decode(JsonInstanceBinary, [{return_maps, false}]),
```

Comment:
It might be an ok change, and I don't have all the details in my head at this very moment, but why did you decide to remove return_maps=false ?

Reply:
Since there's the "no macros in dynamic calls" to fix (see the CI run), and I guess the solution will be to create a function for encapsulation, maybe we just keep the return_maps=false

Reply:
I removed it because `json` does not support it, and I followed the logic and manually verified it is not needed (anymore) 🤔 

That CI error, huh, elvis. Will fix 👌🏽

Reply:
Pushed a fix :)

Reply:
Sorry, made a typo on the fix, pushed again 😅